README.md

```diff
@@ -40,9 +40,9 @@ To enable Liberty Maven Plugin in your project add the following to your `pom.xm
         <plugins>
             <!-- Enable liberty-maven-plugin -->
             <plugin>
-                <groupId>net.wasdev.wlp.maven.plugins</groupId>
+                <groupId>io.openliberty.tools</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>2.5</version>
+                <version>3.0</version>
```

Comment:
Do we want to have `3.0` here before the 3.0 release?

Reply:
I think so because we are preparing for the 3.0 release..although doc updates are immediate...so interesting question.

Reply:
Should I change it to 3.0-M2 for now?

Reply:
Maybe just this one in README.md and leave the rest as they are.